spec/factories/standards_imports.rb

```diff
@@ -8,6 +8,10 @@
       files { [Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", "files", "docx_file_attachments.docx"))] }
     end
 
+    trait :with_pdf_file_with_attachments do
```

Comment:
Can a PDF file have attachments in the same way a docx file can? If not how do you feel about this name instead? 
```suggestion
    trait :with_pdf_file do
```

Reply:
@jutonz good catch! PDF do not have any attachments